portals/publisher/src/main/webapp/source/src/app/components/Shared/PoliciesUI/AttachedPolicyCard.tsx

```diff
@@ -106,70 +109,153 @@ const AttachedPolicyCardShared: FC<AttachedPolicyCardSharedProps> = ({
         backgroundColor: policyBackgroundColor,
         opacity: isDragging ? 0.5 : 1,
     };
-    return (
-        (<Root>
-            <div
-                ref={setNodeRef}
-                style={style}
-                {...attributes}
-                {...listeners}
-                onClick={handleDrawerOpen}
-                onKeyDown={handleDrawerOpen}
-            >
-                <Tooltip
-                    key={policyObj.id}
-                    title={`${policyObj.displayName} : ${policyObj.version}`}
-                    placement='top'
+
+    if ('listOriginatedFromCommonPolicies' in props) {
+        // Props were passed, use `listOriginatedFromCommonPolicies` and `isApiRevision`
+        let policyType =   "Common Policy";
```

Comment:
Shall we make `Common Policy` and `API Policy` constants?

Reply:
fixed